languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -10291,6 +10291,17 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
                 <example correction="disproportionnée">Il a recours à l'aide <marker>disproportionnés</marker>.</example>
             </rule>
             <rule>
+                <antipattern>
+                    <token postag="D.* p" postag_regexp="yes"/>
+                    <token postag="N.* p" postag_regexp="yes"/>
+                    <token>à</token>
+                    <example>Consultez les aides à la formation accessibles aux particuliers.</example>
```

Comment:
Not sure about this AP here, but let's check the diff tomorrow

Reply:
Yes, we can look at the rule in the diff. I think it will solve some Fps in the diff like:
Les crédits à la consommation classiques
un service à la clientèle complet
Les blessures à la main dues à la découpe d'avocats sont courantes 🥑 😨